jobs/src/main/scala/dcos/metronome/repository/impl/kv/marshaller/JobRunMarshaller.scala

```diff
@@ -89,6 +93,10 @@ object JobRunConversions {
         status = proto.getStatus.toModel,
         createdAt = new DateTime(proto.getCreatedAt, DateTimeZone.UTC),
         completedAt = if (proto.hasFinishedAt) Some(new DateTime(proto.getFinishedAt, DateTimeZone.UTC)) else None,
+        startingDeadline = if (proto.hasStartingDeadlineSeconds)
+        Some(Duration(proto.getStartingDeadlineSeconds, TimeUnit.SECONDS))
+      else
+        None,
```

Comment:
I don't really like the alignment of this code.   It isn't consistent with completedAt.  It is longer (perhaps that is the reason to break to more than 1 line).    Now the `else` is hanging out there and it forces the reader to trace back to find the context.

Reply:
I bet you need to update scalariform. I saw a bunch of these fixed when I upgraded Marathon.

Reply:
I'll do that in a separate PR

Reply:
already done and merged